media/static-bundles.json

```diff
@@ -825,10 +825,18 @@
     },
     {
       "files": [
+        "js/base/uitour-lib.js",
```

Comment:
If you rebase your branch against the latest master then this file is now included globally, so no need to add it again here.